google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/it/BigtableBackupIT.java

```diff
@@ -56,11 +64,11 @@
 
 @RunWith(JUnit4.class)
 public class BigtableBackupIT {
+  @ClassRule public static TestEnvRule testEnvRule = new TestEnvRule();
```

Comment:
I think this integration test is in a weird state: it was written in fork where TestEnvRule didnt exist and uses its own property names (ie. bigtable.adminendpoint vs bigtable.admin-endpoint. So this test should either migrate to the TestEnvRule now or we should stick to the bespoke property names

Reply:
I'll give migrating a try

Reply:
@igorbernstein2 switched over to TestEnvRule, PTAL